### PR TITLE
BugReportForm: Remove badly used `DpiUtil.Scale()`

### PR DESCRIPTION
**Pull request by pmiossec**
Theses usages have no effect as the value should be re-affected.

## Test methodology 

- Manual

## Test environment(s) 

- Git Extensions 33.33.33
- Build 286b49ba21e909d81c8399ffb3ad2745a611c22f
- Git 2.42.0.windows.2
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.21
- DPI 96dpi (no scaling)
- Portable: False

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by mstv — COMMENTED** [review]
I don't know much about scaling - so also not whether 02f495ef7cb776f7b00a0c267b2dba156ea4d977 would be better.

**Comment by pmiossec** [comment]
> I don't know much about scaling - so also not whether [02f495e](https://github.com/gitextensions/gitextensions/commit/02f495ef7cb776f7b00a0c267b2dba156ea4d977) would be better.

The 2 current lines are doing nothing because the new size calculated are not assigned to the property.
So we have 2 options:
* remove (to keep current behavior)
* try to fix because we know what was the intention

I choose the 1st one for the main  reason that I'm not able to test when scaling is applied and no one complained on this rarely used form and also because if these 2 buttons should be scaled, I don't know why the 2 others 'Ignore' and 'Quit' are not.

So, I let the fix for someone that is able to fix and test it (if it really matters)

File: BugReporter/BugReportForm.cs [hunk 1]
```diff
@@ -56,8 +56,6 @@ public BugReportForm()
             exceptionTypeLabel.Image = DpiUtil.Scale(exceptionTypeLabel.Image);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
-            DpiUtil.Scale(sendAndQuitButton.MinimumSize);
-            DpiUtil.Scale(btnCopy.MinimumSize);
```
Comment (RussKie) [inline BugReporter/BugReportForm.cs:60]: I think these either have to be
```suggestion
            sendAndQuitButton.MinimumSize = DpiUtil.Scale(sendAndQuitButton.MinimumSize);
            btnCopy.MinimumSize = DpiUtil.Scale(btnCopy.MinimumSize);
```
or
```suggestion
            DpiUtil.Scale(ref sendAndQuitButton.MinimumSize);
            DpiUtil.Scale(ref btnCopy.MinimumSize);
```

**Review by RussKie — COMMENTED** [review]
Could you please help me understand why you want to remove these instead of fixing?

Admittedly, these two properties should be scaled by Windows Forms since .NET 7 (https://github.com/dotnet/winforms/pull/7467), but I'd rather wait until we update to .NET 8 and confirm before removing.

**Comment by pmiossec** [comment]
> Could you please help me understand why you want to remove these instead of fixing?

@RussKie I thought I explained it well with my previous comment.
I had this fix you proposed since a long time in my personal branch and finally decided to spend some time on it to test it and see if everything is good.
As I said the reason that made that I preferred to keep the actual behavior instead of following the initial intention and fixing it is that I can't test it (I don't have a hdpi screen and playing with display screen scaling change nothing) AND looking at the code more carefully I saw that if I choose to fix it, there are 2 others control 'Ignore' and 'Quit' that should be handled. Once again, I can't test the fixes on all controls so I preferred the change that kept the current behavior.

**As I can't test changes related on scaling**, I won't spend more time on this PR as I can't see what else I could do.
But like you I prefer if someone can do and test the proper fix but he will have to handle also `IgnoreButton` and `quitButton`. And I will happily close this PR.

File: BugReporter/BugReportForm.cs [hunk 2]
```diff
@@ -56,8 +56,6 @@ public BugReportForm()
             exceptionTypeLabel.Image = DpiUtil.Scale(exceptionTypeLabel.Image);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
```
[the remark this answers is not in the document]
Comment (pmiossec) [reply BugReporter/BugReportForm.cs]: 👓 Fixed.

**Comment by gerhardol** [comment]
Merge this now and take updates in a follow up - if someone investigates?

**Comment by RussKie** [comment]
Let's see if it's needed at all with .net8

**Comment by pmiossec** [comment]
> Let's see if it's needed at all with .net8

The code is wrong and is doing nothing so it could be removed without problem by merging this PR.
Thanks to @mstv eagle eye, it even fix another original typo.

So I think you could definitively merge this PR and later have a look if good scaling handling is required with .net8...

**Review by mstv — APPROVED** [review]
have not run

**Comment by pmiossec** [comment]
Merging as it is introducing no behavior change on the line deleted (just code clean up due to bad api use).
It's just fixing one typo in the end...

**Pull request merged**